doc/classes/@GlobalScope.xml

```diff
@@ -176,6 +176,18 @@
 				Returns the point at the given [param t] on a one-dimensional [url=https://en.wikipedia.org/wiki/B%C3%A9zier_curve]Bézier curve[/url] defined by the given [param control_1], [param control_2], and [param end] points.
 			</description>
 		</method>
+		<method name="blend_from_angle">
+			<return type="float" />
+			<param index="0" name="rest" type="float" />
+			<param index="1" name="from" type="float" />
+			<param index="2" name="to" type="float" />
+			<param index="3" name="weight" type="float" />
+			<description>
+				Returns the blended value of the [param from] and [param to] angles with respect to the [param rest], but does not cross over the inverted axis [param rest].
+				This method does not respect the shortest path like [method lerp_angle], but instead does not jump values even if more than one interpolation is stacked unless the same [param rest] is passed to them.
+				This is useful for preventing bone penetration into the character's body when you rotate a [Bone2D].
+			</description>
+		</method>
```

Comment:
I think we need to discuss with other maintainers whether this makes sense to implement as a global scope function like this. It seems highly specific, as the comment points out too, so maybe it should be a helper in a more specific class. But if there are valid use cases outside of bone rotations it can make sense as a global utility function.

Reply:
I think the most common use will be within a custom SkeletonModifier3D in GDScript.

I don't strongly disagree with making them static functions outside of global scope, e.g. Skeleton3D or Skeleton2D, but these formulas are applied to other classes than those in AnimationTree's blending process.

So I can't say if that is the right direction to go, since it would require including Skeleton2D just for that purpose in the AnimationTree.